go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/estafette/estafette-k8s-node-compactor
```

Comment:
I did that already, and pushed the vendor folder to master (https://github.com/estafette/estafette-k8s-node-compactor/tree/master/vendor) so that it doesn't show up in the PR.